package.json

```diff
@@ -27,24 +27,38 @@
   "engines": {
     "node": ">=4.0.0"
   },
+  "main": "./dist/index.js",
+  "typings": "./dist/index.d.ts",
+  "files": [
+    "LICENSE",
+    "README.md",
+    "dist/index.js",
+    "dist/index.d.ts"
+  ],
   "scripts": {
-    "pretest": "standard --fix",
-    "test": "mocha"
-  },
-  "devDependencies": {
-    "connect": "^3.7.0",
-    "mocha": "^6.1.4",
-    "standard": "^12.0.1",
-    "supertest": "^4.0.2"
-  },
-  "standard": {
-    "globals": [
-      "describe",
-      "beforeEach",
-      "it"
-    ]
+    "pretest": "npm run lint",
+    "prepublishOnly": "npm run build",
+    "lint": "eslint --fix '**/*.ts'",
+    "test": "jest --config test/jest-config.json",
+    "clean": "rm -rf dist",
+    "build": "npm run clean && tsc"
   },
   "dependencies": {
     "depd": "2.0.0"
+  },
+  "devDependencies": {
```

Comment:
`npm outdated` showed that a few of these deps were out of date. Could you update @types/jest, @typescript-eslint/eslint-plugin, @typescript-eslint/parser, and typescript to their latest versions?

`npm install @types/jest@latest @typescript-eslint/eslint-plugin/latest @typescript-eslint/parser@latest typescript@latest --save-dev` should do it.